app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -235,23 +333,38 @@ class Contact extends React.Component {
                   this.state.current.patient.preferred_contact_method === 'Telephone call' ||
                   this.state.current.patient.preferred_contact_method === 'SMS Text-message' ||
                   this.state.current.patient.preferred_contact_method === 'E-mailed Web Link') && (
-                  <Form.Group as={Col} lg="12" controlId="preferred_contact_time">
+                  <Form.Group as={Col} lg="12" id="preferred_contact_time_wrapper" controlId="preferred_contact_time">
                     <Form.Label className="input-label">
                       PREFERRED CONTACT TIME{schema?.fields?.preferred_contact_time?._exclusive?.required && ' *'}
                       <InfoTooltip tooltipTextKey="preferredContactTime" location="right"></InfoTooltip>
                     </Form.Label>
-                    <Form.Control
-                      isInvalid={this.state.errors['preferred_contact_time']}
-                      as="select"
-                      size="lg"
-                      className="form-square"
-                      value={this.state.current.patient.preferred_contact_time || ''}
-                      onChange={this.handleChange}>
-                      <option></option>
-                      <option>Morning</option>
-                      <option>Afternoon</option>
-                      <option>Evening</option>
-                    </Form.Control>
+                    <Select
+                      inputId="preferred_contact_time-select"
+                      name="preferred_contact_time"
+                      value={{
+                        label:
+                          customPreferredContactTimeOptions[this.state.current.patient.preferred_contact_time] ||
+                          this.state.current.patient.preferred_contact_time ||
+                          '',
+                        value: this.state.current.patient.preferred_contact_time || '',
+                      }}
+                      placeholder=""
+                      options={basicPreferredContactTimeOptions.concat(
```

Comment:
done: https://github.com/SaraAlert/SaraAlert/pull/1019/commits/826f9c6d68b3f7ee0337b7f05452fb06099c557a

Reply:
I think a blank dropdown option is preferable to the X button to clear it, to be consistent with the other dropdowns. My only issue with it before was that the blank option was too narrow.

Reply:
The other dropdowns also have it so the top of the dropdown appears on top of where the value is displayed rather than below it, e.g.
![image](https://user-images.githubusercontent.com/8235974/130522845-e46c60d9-00e1-4d97-b451-21b349362cc0.png)

Reply:
Also realizing now the font and spacing look different

Reply:
oh, so most of the drop-downs in the app (including the example above) uses the bootstrap form select component whereas this one uses the React Select component (primary/secondary language are examples of these, which is what I copied this style from for removing the blank option and adding an "X" instead)

Reply:
Can we include the blank option and just tweak the styling to look more similar to the other dropdowns? Doesn't have to be perfect, but making it more similar would be nice if it doesn't require bending over backwards to do.

Reply:
I would say just making the height of the blank option consistent with the other options would be the most important, since that actually affects the UX in that it makes it clearer to the user that blank is in fact an option. After that, font/spacing, and then least important the alignment of the dropdown over the input box.

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/1019/commits/89cb43e5488340368a915af9b23c8ad651d0e536

Reply:
Preferred Contact Time isn't rendering now for me
![image](https://user-images.githubusercontent.com/8235974/130631525-60621592-eae9-4a70-8c0b-42398f7dd9cc.png)

Reply:
Not sure why, not seeing any JS errors

Reply:
I think you have to change the preferred reporting method to something other than unknown/opt-out

Reply:
Oh wow never realized that. Thanks.